services/movie_session.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+from django.db.models.functions import TruncDate
+
+import init_django_orm  # noqa: F401
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
```

Comment:
Ensure that `movie_show_time` is a valid `datetime` object before passing it to `MovieSession.objects.create`. Consider parsing the string to a `datetime` object within this function to avoid potential errors.